apkid/rules/dex/obfuscators.yara

```diff
@@ -169,4 +169,17 @@ rule arxan_multidex : obfuscator
     not arxan
 }
 
+rule allatori_demo : obfuscator
```

Comment:
This rule is also added in https://github.com/rednaga/APKiD/pull/88/files so I don't think I can merge both of these. Could you remove the allatori rule from both this PR and https://github.com/rednaga/APKiD/pull/88/files and add it to a separate PR?